packages/neptune-extension/src/common/components/input/Input.tsx

```diff
@@ -0,0 +1,64 @@
+// Lib
+import React from 'react';
+import { bemBlock } from 'common/utils/bem';
+
+// Module
+import './Input.less';
+
+interface InputProps {
+  className?: string
+  disabled?: boolean
+  error?: boolean
+  selectOnFocus?: boolean
+  variant?: 'small' | 'big'
+  onFocus?: (ev?: React.FocusEvent<HTMLInputElement>) => void
```

Comment:
You can change it to:
` onFocus?: React.FocusEventHandler<HTMLInputElement>`

I think it is more generic.

Reply:
I agree